recipe/meta.yaml

```diff
@@ -13,8 +13,9 @@ source:
     {{ hash_type }}: {{ hash_val }}
 
 build:
-    number: 0
-    script: python setup.py install
+    noarch: python
+    number: 1
+    script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
     build:
```

Comment:
@CJ-Wright, no pip here

Reply:
Interesting, that's a new one! I think this is another spacing issue (the `build` is indented by 4 which we aren't expecting).